blog/2022-03-28-the-unreasonable-power-of-const-assertions.mdx

```diff
@@ -0,0 +1,218 @@
+
+---
+authors: sudo-jaa
+title: 👾Taming Type Inferenece
+description: as const assertions
+slug: taming-type-inference
+tags: [Typescript, Programming, Types, Type Inference]
+---
+
+
+**We've all been there.** It is an unholy hour of the early morning. Your mind has deteriorated to little more than a sludge of grey matter. Your eyes are bloodshot from
+your abiding caffiene dependency. Your hands have long since gnarled from the lashes of carpal tunnel and the endless droning your blue-switched mechanical keyboard.
+The abandoned church from across the street tolls its cacophanous toll for what feels like the thousandth time tonight.
+"Why?" you whimper to yourself.
+"Why the heck won't my Typescript compile?".
```

Comment:
And you say you are not good at writing. You and @dr-farr should be novelists. 

You might want to see Darren comments on Dan post about this inadvertently making it seems that VH make you work all hours of the day 🤣

Reply:
Dan inspires me. I wanna be more like Dan

Reply:
blue switches? Doesn't that drive you crazy?